spec/hedgelog_spec.rb

```diff
@@ -1,11 +1,300 @@
 require 'spec_helper'
+require 'timecop'
+require 'benchmark'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.